x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -1126,7 +1172,7 @@ export class EndpointDocGenerator {
     return [...this.randomNGenerator(255, 6)].map((x) => x.toString(16)).join('-');
   }
 
-  private randomIP(): string {
+  public randomIP(): string {
```

Comment:
why was this made `public`?

Reply:
Per our discussion, this is supposed to be used in a test, but the test has gone missing.

Reply:
tests got found